CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
-## 0.38.0
+## [0.38.1] 19th March 2025
+
+- chore: Export event localizations (Krille)
+- chore: Remove unnecessary template (Krille)
+- feat: Add command to upgrade room (Krille)
+- feat: Add parameter for allow condition for restricted rooms (Krille)
+- feat: allow skipping existing chat when calling startDirectChat (td)
+- feat: export timeline events (Karthikeyan S)
+- feat: upload code coverage to codecov (td)
+- fix: incorrect filename when caption is used in body of event (Karthikeyan S)
+- fix: incorrect mimeType for files when downloading (Karthikeyan S)
+- fix: Older state events overwrite newer ones on fetching history (Krille)
+- fix: room prev_batch set incorrectly (Karthikeyan S)
+- fix: room prev_batch set incorrectly follow-up (Karthikeyan S)
+- fix: set prev_batch first time from sync response correctly (Karthikeyan S)
+- fix: set prev_batch when seen in sync (td)
```

Comment:
Can we combine these 4 changelog entries to one? I don't think the follow ups are interesting